Tests fail due to missing fixture

### Operating System

_No response_

### Programming Language

_No response_

### CCXT Version

_No response_

### Description

```
___________________________________________________________ ERROR at setup of test_fetch_borrow_interest[trio] ____________________________________________________________
[gw0] freebsd15 -- Python 3.12.13 /usr/local/bin/python3.12
file /usr/ports/finance/py-ccxt/work-py312/ccxt-4.5.70/python/ccxt/test/exchange/async/test_fetch_borrow_interest.py, line 18
  async def test_fetch_borrow_interest(exchange, skipped_properties, code, symbol):
      method = 'fetchBorrowInterest'
      borrow_interest = await exchange.fetch_borrow_interest(code, symbol)
      test_shared_methods.assert_non_emtpy_array(exchange, skipped_properties, method, borrow_interest, code)
      for i in range(0, len(borrow_interest)):
          test_borrow_interest(exchange, skipped_properties, method, borrow_interest[i], code, symbol)
      return True
E       fixture 'exchange' not found
>       available fixtures: _asyncio_loop_factory, _class_scoped_runner, _function_scoped_runner, _module_scoped_runner, _package_scoped_runner, _session_faker, _session_scoped_runner, anyio_backend, anyio_backend_name, anyio_backend_options, assert_all_responses_were_requested, cache, capfd, capfdbinary, caplog, capsys, capsysbinary, capteesys, class_based_httpbin, class_based_httpbin_secure, class_mocker, cov, doctest_namespace, event_loop_policy, faker, faker_seed, free_tcp_port, free_tcp_port_factory, free_udp_port, free_udp_port_factory, httpbin, httpbin_both, httpbin_ca_bundle, httpbin_secure, httpserver, httpserver_ipv4, httpserver_ipv6, httpserver_listen_address, httpserver_ssl_context, httpx_mock, make_httpserver, make_httpserver_ipv4, make_httpserver_ipv6, mocker, module_mocker, monkeypatch, no_cover, non_mocked_hosts, package_mocker, pytestconfig, record_property, record_testsuite_property, record_xml_attribute, recwarn, respx_mock, session_mocker, socket_disabled, socket_enabled, subtests, testrun_uid, time_machine, tmp_path, tmp_path_factory, tmpdir, tmpdir_factory, unused_tcp_port, unused_tcp_port_factory, unused_udp_port, unused_udp_port_factory, weave, worker_id
>       use 'pytest --fixtures [testpath]' for help on them.

/usr/ports/finance/py-ccxt/work-py312/ccxt-4.5.70/python/ccxt/test/exchange/async/test_fetch_borrow_interest.py:18
_________________________________________________________________ ERROR at setup of test_borrow_interest __________________________________________________________________
[gw0] freebsd15 -- Python 3.12.13 /usr/local/bin/python3.12
file /usr/ports/finance/py-ccxt/work-py312/stage/usr/local/lib/python3.12/site-packages/ccxt/test/exchange/base/test_borrow_interest.py, line 17
  def test_borrow_interest(exchange, skipped_properties, method, entry, requested_code, requested_symbol):
E       fixture 'exchange' not found
>       available fixtures: _asyncio_loop_factory, _class_scoped_runner, _function_scoped_runner, _module_scoped_runner, _package_scoped_runner, _session_faker, _session_scoped_runner, anyio_backend, anyio_backend_name, anyio_backend_options, assert_all_responses_were_requested, cache, capfd, capfdbinary, caplog, capsys, capsysbinary, capteesys, class_based_httpbin, class_based_httpbin_secure, class_mocker, cov, doctest_namespace, event_loop_policy, faker, faker_seed, free_tcp_port, free_tcp_port_factory, free_udp_port, free_udp_port_factory, httpbin, httpbin_both, httpbin_ca_bundle, httpbin_secure, httpserver, httpserver_ipv4, httpserver_ipv6, httpserver_listen_address, httpserver_ssl_context, httpx_mock, make_httpserver, make_httpserver_ipv4, make_httpserver_ipv6, mocker, module_mocker, monkeypatch, no_cover, non_mocked_hosts, package_mocker, pytestconfig, record_property, record_testsuite_property, record_xml_attribute, recwarn, respx_mock, session_mocker, socket_disabled, socket_enabled, subtests, testrun_uid, time_machine, tmp_path, tmp_path_factory, tmpdir, tmpdir_factory, unused_tcp_port, unused_tcp_port_factory, unused_udp_port, unused_udp_port_factory, weave, worker_id
>       use 'pytest --fixtures [testpath]' for help on them.

/usr/ports/finance/py-ccxt/work-py312/stage/usr/local/lib/python3.12/site-packages/ccxt/test/exchange/base/test_borrow_interest.py:17
__________________________________________________________ ERROR at setup of test_fetch_borrow_interest[asyncio] __________________________________________________________
[gw0] freebsd15 -- Python 3.12.13 /usr/local/bin/python3.12
file /usr/ports/finance/py-ccxt/work-py312/ccxt-4.5.70/python/ccxt/test/exchange/async/test_fetch_borrow_interest.py, line 18
  async def test_fetch_borrow_interest(exchange, skipped_properties, code, symbol):
      method = 'fetchBorrowInterest'
      borrow_interest = await exchange.fetch_borrow_interest(code, symbol)
      test_shared_methods.assert_non_emtpy_array(exchange, skipped_properties, method, borrow_interest, code)
      for i in range(0, len(borrow_interest)):
          test_borrow_interest(exchange, skipped_properties, method, borrow_interest[i], code, symbol)
      return True
E       fixture 'exchange' not found
>       available fixtures: _asyncio_loop_factory, _class_scoped_runner, _function_scoped_runner, _module_scoped_runner, _package_scoped_runner, _session_faker, _session_scoped_runner, anyio_backend, anyio_backend_name, anyio_backend_options, assert_all_responses_were_requested, cache, capfd, capfdbinary, caplog, capsys, capsysbinary, capteesys, class_based_httpbin, class_based_httpbin_secure, class_mocker, cov, doctest_namespace, event_loop_policy, faker, faker_seed, free_tcp_port, free_tcp_port_factory, free_udp_port, free_udp_port_factory, httpbin, httpbin_both, httpbin_ca_bundle, httpbin_secure, httpserver, httpserver_ipv4, httpserver_ipv6, httpserver_listen_address, httpserver_ssl_context, httpx_mock, make_httpserver, make_httpserver_ipv4, make_httpserver_ipv6, mocker, module_mocker, monkeypatch, no_cover, non_mocked_hosts, package_mocker, pytestconfig, record_property, record_testsuite_property, record_xml_attribute, recwarn, respx_mock, session_mocker, socket_disabled, socket_enabled, subtests, testrun_uid, time_machine, tmp_path, tmp_path_factory, tmpdir, tmpdir_factory, unused_tcp_port, unused_tcp_port_factory, unused_udp_port, unused_udp_port_factory, weave, worker_id
>       use 'pytest --fixtures [testpath]' for help on them.

/usr/ports/finance/py-ccxt/work-py312/ccxt-4.5.70/python/ccxt/test/exchange/async/test_fetch_borrow_interest.py:18
_____________________________________________________________________ ERROR at setup of test_currency _____________________________________________________________________
[gw0] freebsd15 -- Python 3.12.13 /usr/local/bin/python3.12
file /usr/ports/finance/py-ccxt/work-py312/ccxt-4.5.70/python/ccxt/test/exchange/base/test_currency.py, line 18
  def test_currency(exchange, skipped_properties, method, entry):
E       fixture 'exchange' not found
>       available fixtures: _asyncio_loop_factory, _class_scoped_runner, _function_scoped_runner, _module_scoped_runner, _package_scoped_runner, _session_faker, _session_scoped_runner, anyio_backend, anyio_backend_name, anyio_backend_options, assert_all_responses_were_requested, cache, capfd, capfdbinary, caplog, capsys, capsysbinary, capteesys, class_based_httpbin, class_based_httpbin_secure, class_mocker, cov, doctest_namespace, event_loop_policy, faker, faker_seed, free_tcp_port, free_tcp_port_factory, free_udp_port, free_udp_port_factory, httpbin, httpbin_both, httpbin_ca_bundle, httpbin_secure, httpserver, httpserver_ipv4, httpserver_ipv6, httpserver_listen_address, httpserver_ssl_context, httpx_mock, make_httpserver, make_httpserver_ipv4, make_httpserver_ipv6, mocker, module_mocker, monkeypatch, no_cover, non_mocked_hosts, package_mocker, pytestconfig, record_property, record_testsuite_property, record_xml_attribute, recwarn, respx_mock, session_mocker, socket_disabled, socket_enabled, subtests, testrun_uid, time_machine, tmp_path, tmp_path_factory, tmpdir, tmpdir_factory, unused_tcp_port, unused_tcp_port_factory, unused_udp_port, unused_udp_port_factory, weave, worker_id
>       use 'pytest --fixtures [testpath]' for help on them.

/usr/ports/finance/py-ccxt/work-py312/ccxt-4.5.70/python/ccxt/test/exchange/base/test_currency.py:18
___________________________________________________________________ ERROR at setup of test_ledger_entry ___________________________________________________________________
[gw1] freebsd15 -- Python 3.12.13 /usr/local/bin/python3.12
file /usr/ports/finance/py-ccxt/work-py312/stage/usr/local/lib/python3.12/site-packages/ccxt/test/exchange/base/test_ledger_entry.py, line 17
  def test_ledger_entry(exchange, skipped_properties, method, entry, requested_code, now):
E       fixture 'exchange' not found
>       available fixtures: _asyncio_loop_factory, _class_scoped_runner, _function_scoped_runner, _module_scoped_runner, _package_scoped_runner, _session_faker, _session_scoped_runner, anyio_backend, anyio_backend_name, anyio_backend_options, assert_all_responses_were_requested, cache, capfd, capfdbinary, caplog, capsys, capsysbinary, capteesys, class_based_httpbin, class_based_httpbin_secure, class_mocker, cov, doctest_namespace, event_loop_policy, faker, faker_seed, free_tcp_port, free_tcp_port_factory, free_udp_port, free_udp_port_factory, httpbin, httpbin_both, httpbin_ca_bundle, httpbin_secure, httpserver, httpserver_ipv4, httpserver_ipv6, httpserver_listen_address, httpserver_ssl_context, httpx_mock, make_httpserver, make_httpserver_ipv4, make_httpserver_ipv6, mocker, module_mocker, monkeypatch, no_cover, non_mocked_hosts, package_mocker, pytestconfig, record_property, record_testsuite_property, record_xml_attribute, recwarn, respx_mock, session_mocker, socket_disabled, socket_enabled, subtests, testrun_uid, time_machine, tmp_path, tmp_path_factory, tmpdir, tmpdir_factory, unused_tcp_port, unused_tcp_port_factory, unused_udp_port, unused_udp_port_factory, weave, worker_id
>       use 'pytest --fixtures [testpath]' for help on them.

/usr/ports/finance/py-ccxt/work-py312/stage/usr/local/lib/python3.12/site-packages/ccxt/test/exchange/base/test_ledger_entry.py:17
________________________________________________________________ ERROR at setup of test_fetch_ledger[trio] ________________________________________________________________
[gw1] freebsd15 -- Python 3.12.13 /usr/local/bin/python3.12
file /usr/ports/finance/py-ccxt/work-py312/ccxt-4.5.70/python/ccxt/test/exchange/async/test_fetch_ledger.py, line 18
  async def test_fetch_ledger(exchange, skipped_properties, code):
      method = 'fetchLedger'
      items = await exchange.fetch_ledger(code)
      test_shared_methods.assert_non_emtpy_array(exchange, skipped_properties, method, items, code)
      now = exchange.milliseconds()
      for i in range(0, len(items)):
          test_ledger_entry(exchange, skipped_properties, method, items[i], code, now)
      test_shared_methods.assert_timestamp_order(exchange, method, code, items)
      return True
E       fixture 'exchange' not found
>       available fixtures: _asyncio_loop_factory, _class_scoped_runner, _function_scoped_runner, _module_scoped_runner, _package_scoped_runner, _session_faker, _session_scoped_runner, anyio_backend, anyio_backend_name, anyio_backend_options, assert_all_responses_were_requested, cache, capfd, capfdbinary, caplog, capsys, capsysbinary, capteesys, class_based_httpbin, class_based_httpbin_secure, class_mocker, cov, doctest_namespace, event_loop_policy, faker, faker_seed, free_tcp_port, free_tcp_port_factory, free_udp_port, free_udp_port_factory, httpbin, httpbin_both, httpbin_ca_bundle, httpbin_secure, httpserver, httpserver_ipv4, httpserver_ipv6, httpserver_listen_address, httpserver_ssl_context, httpx_mock, make_httpserver, make_httpserver_ipv4, make_httpserver_ipv6, mocker, module_mocker, monkeypatch, no_cover, non_mocked_hosts, package_mocker, pytestconfig, record_property, record_testsuite_property, record_xml_attribute, recwarn, respx_mock, session_mocker, socket_disabled, socket_enabled, subtests, testrun_uid, time_machine, tmp_path, tmp_path_factory, tmpdir, tmpdir_factory, unused_tcp_port, unused_tcp_port_factory, unused_udp_port, unused_udp_port_factory, weave, worker_id
>       use 'pytest --fixtures [testpath]' for help on them.

/usr/ports/finance/py-ccxt/work-py312/ccxt-4.5.70/python/ccxt/test/exchange/async/test_fetch_ledger.py:18
______________________________________________________________ ERROR at setup of test_fetch_ledger[asyncio] _______________________________________________________________
[gw1] freebsd15 -- Python 3.12.13 /usr/local/bin/python3.12
file /usr/ports/finance/py-ccxt/work-py312/ccxt-4.5.70/python/ccxt/test/exchange/async/test_fetch_ledger.py, line 18
  async def test_fetch_ledger(exchange, skipped_properties, code):
      method = 'fetchLedger'
      items = await exchange.fetch_ledger(code)
      test_shared_methods.assert_non_emtpy_array(exchange, skipped_properties, method, items, code)
      now = exchange.milliseconds()
      for i in range(0, len(items)):
          test_ledger_entry(exchange, skipped_properties, method, items[i], code, now)
      test_shared_methods.assert_timestamp_order(exchange, method, code, items)
      return True
E       fixture 'exchange' not found
>       available fixtures: _asyncio_loop_factory, _class_scoped_runner, _function_scoped_runner, _module_scoped_runner, _package_scoped_runner, _session_faker, _session_scoped_runner, anyio_backend, anyio_backend_name, anyio_backend_options, assert_all_responses_were_requested, cache, capfd, capfdbinary, caplog, capsys, capsysbinary, capteesys, class_based_httpbin, class_based_httpbin_secure, class_mocker, cov, doctest_namespace, event_loop_policy, faker, faker_seed, free_tcp_port, free_tcp_port_factory, free_udp_port, free_udp_port_factory, httpbin, httpbin_both, httpbin_ca_bundle, httpbin_secure, httpserver, httpserver_ipv4, httpserver_ipv6, httpserver_listen_address, httpserver_ssl_context, httpx_mock, make_httpserver, make_httpserver_ipv4, make_httpserver_ipv6, mocker, module_mocker, monkeypatch, no_cover, non_mocked_hosts, package_mocker, pytestconfig, record_property, record_testsuite_property, record_xml_attribute, recwarn, respx_mock, session_mocker, socket_disabled, socket_enabled, subtests, testrun_uid, time_machine, tmp_path, tmp_path_factory, tmpdir, tmpdir_factory, unused_tcp_port, unused_tcp_port_factory, unused_udp_port, unused_udp_port_factory, weave, worker_id
>       use 'pytest --fixtures [testpath]' for help on them.

/usr/ports/finance/py-ccxt/work-py312/ccxt-4.5.70/python/ccxt/test/exchange/async/test_fetch_ledger.py:18
```

Tarball is downloaded from PYOPI.
Is conftest.py missing?


Version: 4.5.70
Python: 3.12
FreeBSD 15.1

--- a/python/ccxt/test/conftest.py
+++ b/python/ccxt/test/conftest.py
@@ -1,0 +1,11 @@
+# This directory contains ccxt's own integration test harness, driven by
+# ccxt/test/tests_init.py (see the repository CI), not by pytest.
+# The test functions here are parameterized by the harness
+# (exchange, skipped_properties, ...) and run against live exchange APIs,
+# so collecting them with pytest fails with "fixture 'exchange' not found"
+# and they would not be runnable offline in any case.
+# This guard makes pytest skip the whole tree gracefully, which matters for
+# downstream packagers (FreeBSD ports, Debian, nixpkgs, conda-forge) that
+# reflexively run pytest over installed site-packages.
+# See https://github.com/ccxt/ccxt/issues/29383
+collect_ignore_glob = ['*']

